src/pages/docs/chat/api/javascript/message.mdx

```diff
@@ -0,0 +1,393 @@
+| `deletedAt` | The timestamp at which the message was deleted. | Date \| `undefined` |
+| `deletedBy` | The clientId of the user who deleted the message. | String \| `undefined` |
+| `isDeleted` | Indicates if the message has been deleted. | Boolean |
+| `isUpdated` | Indicates if the message has been updated. | Boolean |
+| `updatedAt` | The timestamp at which the message was updated. | Date \| `undefined` |
+| `updatedBy` | The clientId of the user who updated the message. | String \| `undefined` |
+
+---
+
+## Properties
+
+It has the following properties:
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| `action` | The action type of the message. This can be used to determine if the message was created, updated, or deleted. | [`ChatMessageAction`](#chatmessageaction) |
+| `clientId` | The clientId of the user who created the message. | String |
+| `createdAt` | The timestamp at which the message was created. | Date |
+| `headers` | The headers of a chat message. Headers enable attaching extra info to a message, which can be used for various features such as linking to a relative point in time of a livestream video or flagging this message as important or pinned. This value is always set. If there are no headers, this is an empty object. Do not use the headers for authoritative information. There is no server-side validation. When reading the headers, treat them like user input. | [`MessageHeaders`](#messageheaders) |
+| `metadata` | The metadata of a chat message. Allows for attaching extra info to a message, which can be used for various features such as animations, effects, or simply to link it to other resources such as images, relative points in time, etc. This value is always set. If there is no metadata, this is an empty object. Do not use metadata for authoritative information. There is no server-side validation. When reading the metadata treat it like user input. | [`MessageMetadata`](#messagemetadata) |
+| `operation` | *Optional* The details of the operation that modified the message. This is only set for update and delete actions. It contains information about the operation: the clientId of the user who performed the operation, a description, and metadata. | [`Operation`](#operation) |
+| `reactions` | The reactions summary for this message. | [`MessageReactions`](#messagereactions) |
+| `serial` | The unique identifier of the message. | String |
+| `text` | The text of the message. | String |
+| `timestamp` | The timestamp at which this version was updated, deleted, or created. | Date |
+| `version` | A unique identifier for the latest version of this message. | String |
+
+---
+
+### ChatMessageAction
+
+An enum that represents the types of action of a message.
+
+It has the following members:
+
+| Member | Description |
+| ------ | ----------- |
+| `Create` | The message was created. |
+| `Update` | The message was updated. |
+| `Delete` | The message was deleted. |
+
+### MessageHeaders | Headers
```

Comment:
Ignore `Headers` and `Metadata` - the fact that MessageHeaders currently is an alias for that type shouldn't appear in the API refs imo